src/sentry/static/sentry/app/views/taskList/index.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString';
+import Tasks from 'app/views/taskList/tasks';
+import {connect} from 'react-redux';
+import {tagsGet} from 'app/redux/actions/tag';
+
+class TaskList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    const {getTags} = this.props;
+    getTags();
```

Comment:
I wonder why the component does this?

Reply:
Well, eventually we will want filtering of user tasks by tags. We already have a redux action to fetch tags so that's just a step in that direction.